app/Models/GameSet.php

```diff
@@ -78,6 +78,20 @@ public static function boot()
                     ])
                     ->event('pivotAttached')
                     ->log('pivotAttached');
+
+                // Log the attachment on each game model.
+                foreach ($attachedGames as $game) {
+                    $attribute = $model->type === GameSetType::Hub ? 'hubs' : 'similarGames';
+
+                    activity()->causedBy($user)->performedOn($game)
+                        ->withProperty('old', [$attribute => null])
+                        ->withProperty('attributes', [$attribute => [
+                            'id' => $model->id,
+                            'title' => $model->title,
+                        ]])
+                        ->event('pivotAttached')
+                        ->log('pivotAttached');
+                }
```

Comment:
It probably doesn't matter, but multiple add/removes in a single commit are creating separate activity records:
![image](https://github.com/user-attachments/assets/6aeed924-1df2-425e-87bf-66ad808272f1)

Reply:
Probably not a huge deal. If this becomes cumbersome, we can revisit it later (or alternatively come up with some kind of squashing treatment when we display these in the UI).